apis/compute/v1beta1/forwardingrule_types.go

```diff
@@ -151,6 +151,18 @@ type ComputeForwardingRuleSpec struct {
 	// +optional
 	Description *string `json:"description,omitempty"`
 
+	// The GCP labels assigned to this resource.
+	//
+	// GCPLabel keys must be between 1 and 63 characters long, have a UTF-8 encoding
+	// of maximum 128 bytes, and must conform to the following PCRE regular
+	// expression: `[\p{Ll}\p{Lo}][\p{Ll}\p{Lo}\p{N}_-]{0,62}`
+	//
+	// GCPLabel values must be between 0 and 63 characters long, have a UTF-8
+	// encoding of maximum 128 bytes, and must conform to the following PCRE
+	// regular expression: `[\p{Ll}\p{Lo}\p{N}_-]{0,63}`
+	//
+	// No more than 64 labels can be assigned to a given resource.
+	GCPLabels map[string]string `json:"gcpLabels,omitempty"`
```

Comment:
I think there's a misunderstanding about the spec.label names. Let's sync offline